Shared/Samples/Edit features using feature forms/EditFeaturesUsingFeatureFormsView.swift

```diff
@@ -103,102 +103,63 @@ struct EditFeaturesUsingFeatureFormsView: View {
                         .padding(8)
                         .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
                 }
-                .floatingPanel(isPresented: $isShowingFeatureForm) { [featureForm] in
+                .sheet(isPresented: $isShowingFeatureForm) {
                     if let featureForm {
-                        VStack {
-                            featureFormToolbar
-                            
-                            // Displays the feature form using the toolkit component.
-                            FeatureFormView(featureForm: featureForm)
-                                .padding(.horizontal)
-                                .task {
-                                    defer { hasEdits = false }
-                                    
-                                    for await hasEdits in featureForm.$hasEdits {
-                                        self.hasEdits = hasEdits
-                                    }
+                        // Displays the feature form using the toolkit component.
+                        FeatureFormView(root: featureForm, isPresented: $isShowingFeatureForm)
+                            .onFormEditingEvent { event in
+                                switch event {
+                                case .discardedEdits:
+                                    endEditing()
```

Comment:
I used the built-in save and discard buttons. Since the discard button discard the edits itself, I got rid of the alert.